src/database/schema/ChannelListSchema.ts

```diff
@@ -410,6 +414,7 @@ class ChannelList implements BaseDbSchema {
       createdAt: json.created_at,
       rawJson: jsonParsed,
       members: json.members,
+      memberUsers: json.memberUsers,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `fromJSON` static method has been updated to parse `memberUsers` from the JSON object. This is consistent with the new property added to the class.
```diff
+      memberUsers: json.memberUsers,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->